runtime/src/main/resources/packaged/configuration.yml

```diff
@@ -1,7 +1,10 @@
+flux.Shard.Config:
+  master.slave.pair.list: { "masterSlavePairList" : [ { "master":{ "shardId":1, "ip": "127.0.0.1", "shardKeys": ["0","1","2","3","4","5","6","7","8","9","a","b","c","d","e","f"] },"slave":{ "shardId":2, "ip":"127.0.0.1", "shardKeys": ["0","1","2","3","4","5","6","7","8","9","a","b","c","d","e","f"] }}  ]  }
```

Comment:
1. Instead of keys, take range. 
2. This model looks redundant (shard keys and IDs) and error prone (mentioning master's shard ID 1 and slave's as 2)

ShardConfig: [
  {
    "ShardID": 1,
    "MasterIP": "localhost",
    "SlaveIP": "localhost",
    "StartOfRange": "00",
    "EndOfRange": "80"
  },
  {
    "ShardID": 2,
    "MasterIP": "10.20.30.40",
    "SlaveIP": "10.20.30.50",
    "StartOfRange": "80",
    "EndOfRange": "ff"
  }
]

Start of range is inclusive and End of range is exclusive
basically the keys which go in to first shard is 00 <= key < 80